code/src/components/BarcodeScanner.js

```diff
@@ -39,4 +50,4 @@ export const BarcodeScanner = ({ className, onDetected }) => {
       <div ref={cameraDivRef} className={className} />
     </>
   )
-}
+}
```

Comment:
this **div** is what you could target for the styled components. Instead of styling it in the css.
```
const Scanner = styled.div`
  // the styling you hav in .scanner class now

  video {
    // the styling you have in .scanner video now
  }
  
  canvas {
    // the styling you have in .scanner canvas now
  }
`
```
Let me know if this wasn't explanation enough